src/Formatter/Formatter.php

```diff
@@ -197,9 +197,6 @@ public function getJs()
         $configurator->enableJavaScript();
         $configurator->javascript->exportMethods = ['preview'];
 
-        return $configurator->finalize([
-            'returnParser' => false,
-            'returnRenderer' => false
-        ])['js'];
+        return $configurator->finalize()['js'];
```

Comment:
I think this can be optimized.
`enableJavaScript()` can moved to `getConfigurator()` and `getComponent()` can be used here, e.g:

```
return $this->getComponent('js');
```

but as `getComponent()` results will cache I'm not sure if it's good idea or not

Reply:
I sounds like a reasonable thing to do.